mimic/model/nova_image_collection.py

```diff
@@ -6,32 +6,23 @@
 
 from characteristic import attributes, Attribute
 from json import dumps
-from mimic.model.rackspace_images import (ImageStore, OnMetalImage)
+from mimic.model.rackspace_images import OnMetalImage
 
 from mimic.model.nova_objects import not_found
 from mimic.canned_responses.mimic_presets import get_presets
 
 
 @attributes(
-    ["tenant_id", "region_name", "clock"])
-class RegionalImageCollection(object):
+    ["tenant_id", "region_name", "clock", "image_store"])
```

Comment:
This adds a new `image_store` attribute, but there's no documentation of what type it is or where it needs to come from.